.github/workflows/README.md

```diff
@@ -99,14 +99,55 @@ graph TD
 - Pushes HTML to gh-pages branch, which is linked to https://docs.opendp.org
 
 
-## Making one-off releases
+## Knobs on the Release Workflow
 
 One-off releases can be made with the
 [`release.yml` workflow](https://github.com/opendp/opendp/actions/workflows/release.yml)
 on github, or with the `gh` command line tool. Parameters:
 
 - **Target channel** controls how the release is tagged, and what semantic version is given to the release. There is a git branch with the same name for each channel.
-- The **sync the Channel from upstream?** checkbox is for when you want to update the `nightly`, `beta` or `stable` branches.
+- The **sync the Channel from upstream?** checkbox is for when you want to update the `nightly`, `beta` `stable` or `dev` branches. This replaces the contents of the respective branch with the branch the workflow is run from ("Use workflow from"). In practice, you'll almost always enable this.
 - Update the **version counter** accordingly when you want to release multiple nightlies or betas in the same day.
 - **Dry runs** get sent to test-pypi, and don't update the docs
 - **Fake** is for developer convenience when debugging CI: it skips compilation and inserts dummy binaries instead
+
+## Making a release
+
+1. Ensure that changelog is up-to-date and version in `main` matches the version you want to release (otherwise see step 3).
```

Comment:
>  1. Ensure that changelog is up-to-date

"Ensure" sounds like we're just checking, while we really need to author the changelog at the start of this process, I think?

```diff
- version in `main`
+ the `VERSION` file on the `main` branch
```